.github/workflows/norm_check.yml

```
@@ -0,0 +1,30 @@
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  norm-check:
+    name: Norm Check
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)